apps/desktop/src/views/SearchView/components/AskUser/AskUserQuestions.vue

```
@@ -0,0 +1,883 @@
+                            :register-item="registerItem"
+                            :role="null"
+                            :is-selected="otherText.length > 0"
+                            :tab-index="-1"
+                            :on-focus-visible="() => (focusedIndex = otherIndex)"
+                            :on-blur-any="
+                                () => {
+                                    if (focusedIndex === otherIndex) focusedIndex = null;
+                                }
+                            "
+                            :on-click="() => otherInputRef?.focus()"
+                            :aria-label="question?.otherPlaceholder ?? 'Describe in your own words'"
```

ESLint

[error] Replace `:aria-label="question?.otherPlaceholder·??·'Describe·in·your·own·words'` with `····:aria-label="⏎····································question?.otherPlaceholder·??·'Describe·in·your·own·words'⏎································`

(prettier/prettier)